torch/_export/functionalize_assertions.py

```diff
@@ -12,7 +14,15 @@
 }
 
 
-def functionalize(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
+@dataclass
+class FunctionalizedGraphModule:
```

Comment:
It should rename it to something else as naming it GraphModule can create a confusion about whether it has same functionality as normal GraphModule.

Reply:
I guess nothing is harder than naming so let me just return a tuple for simplicity :p